mlflow/shap.py

```diff
@@ -32,3 +39,70 @@ def _log_matplotlib_figure(fig, out_file, artifact_path=None):
     """
     with _log_artifact_contextmanager(out_file, artifact_path) as tmp_path:
         fig.savefig(tmp_path)
+
+
+@experimental
+def log_explanation(predict_function, features, artifact_path=None):
+    """
+    Log the following items generated by SHAP.
+
+        - base values
+        - shap values
+        - summary bar plot
+
+    :param predict_function: A function to compute the output of a model.
+    :param features: A matrix of features to compute SHAP values with.
```

Comment:
```suggestion
    :param features: A matrix of features to compute SHAP values with. The provided features should have shape  
     (nsamples, nfeatures), and can be either a numpy array, pandas DataFrame, or list of lists.
```

Is this correct? Basically the idea is to clarify the expected shape/allowed types of `features`, I tried to infer based on https://shap.readthedocs.io/en/latest/generated/shap.summary_plot.html#shap.summary_plot

Could we clarify